backend/src/telegram/models/telegram-subscriber.ts

```diff
@@ -8,9 +8,15 @@ import { Column, Model, Table, DataType } from 'sequelize-typescript'
 export class TelegramSubscriber extends Model<TelegramSubscriber> {
   @Column({
     type: DataType.STRING,
+    primaryKey: true,
+    unique: true,
   })
   phoneNumber!: string
 
-  @Column({ type: DataType.BIGINT })
+  @Column({
```

Comment:
I'm thinking that the constraint on `phoneNumber` might be enough to ensure uniqueness for the `phoneNumber` - `telegramId` mapping.